app/controllers/dashboard_controller.rb

```diff
@@ -1,4 +1,4 @@
-class DashboardController < ActionController::Base
+class DashboardController < ApplicationController
```

Comment:
I guess this is intentional. The frontend is considered a separate app and it is not following the auth mechanism in application controller.